tests/gdb-tests/tests/test_context_commands.py

```diff
@@ -201,21 +201,19 @@ def test_context_backtrace_show_proper_symbol_names(start_binary):
         == "─────────────────────────────────[ BACKTRACE ]──────────────────────────────────"
     )
 
-    assert re.match(r".*0   0x[0-9a-f]+ A::foo\(int, int\)", backtrace[2])
```

Comment:
how important is it to tightly fit this regex to the entire output in a test that is mainly of proper symbol names?

Reply:
The more accurate we are the less regressions we will have :). I am fine with more relaxed assertions: its better to have some tests than none and its hard to test for everything.